release/update-rsync.sh

```diff
@@ -7,22 +7,33 @@
 
 . ~/.server.config || exit $?
 
-cd `dirname $0`/..
+# ensure we have a clean staging directory
```

Comment:
> PR description: Also, briefly add comment documents to indicate the major tasks performed.

This sentence sounds garbled to me. I suggest removing completely or replacing with "Also add comments to document major steps" or something like that.

Reply:
done

Reply:
> Also, briefly add comments to document the major tasks performed.

FWIW, "briefly add comments" still does not mean what I think we want to say here.